Explorer/Assets/DCL/InWorldCamera/InWorldCamera/ScreenshotMetadataBuilder.cs

```diff
@@ -97,7 +97,8 @@ private static string[] FilterNonBaseWearables(IReadOnlyCollection<URN> avatarWe
             return wearables.ToArray();
         }
 
-        internal void FillMetadata(Profile profile, RealmData realm, Vector2Int playerPosition, string sceneName, VisiblePerson[] visiblePeople)
+        internal void FillMetadata(Profile profile, RealmData realm, Vector2Int playerPosition, string sceneName, string placeId,
+            VisiblePerson[] visiblePeople)
```

Comment:
Consider observing a line length that fits into Github's UI. I've measured it to be 105 characters.
![image](https://github.com/user-attachments/assets/38b419fe-0cd3-43bc-90d3-ffc084944462)

Reply:
we can agree with the team on this line break value and put it in the formatting. On the time when we were agreeing, team decided to not enforce line length break